runtime-decompiler/src/main/java/org/jrd/backend/data/ArchiveManagerOptions.java

```diff
@@ -0,0 +1,23 @@
+package org.jrd.backend.data;
+
+import java.util.ArrayList;
+
+public class ArchiveManagerOptions {
+	private static ArrayList<String> extensions = new ArrayList<>();
```

Comment:
Can you please move the default suffxies (from name.endsWith(".zip") || name.endsWith(".jar") || name.endsWith(".war") || name.endsWith(".ear") ) into here? Some static list<string> defaults.

Then I can call form gui getDefaults and you can use defaults.contains  smoothly